setup.py

```diff
@@ -62,11 +62,70 @@
     },
     entry_points=dict(
      console_scripts=[
-     # <hack>
-     ] + ['{}=spinalcordtoolbox.compat.launcher:main'.format(os.path.splitext(x)[0]) for x in os.listdir("scripts") if x.endswith(".py")] + [
-     # </hack>
-      # TODO add proper command-line entry points from refactored code
-      #'sct_deepseg_gm=spinalcordtoolbox.deepseg_gm.__main__:main',
-     ],
+        '{}=spinalcordtoolbox.compat.launcher:main'.format(x) for x in \
+        [
+            'isct_check_detection',
+            'isct_convert_binary_to_trilinear',
+            'isct_minc2volume-viewer',
+            'isct_test_ants',
+            'isct_warpmovie_generator',
+            'msct_multiatlas_seg',
```

Comment:
deprecated (used by sct_segment_graymatter, which is deprecated)

Reply:
but we might want to revive this code at some point, so maybe we can leave the scripts, and just not expose them

Reply:
deleted reference, kept script